### ISSUE TITLE
Please restore NVDARemote support for old NVDA versions (compatibility issue)

### ISSUE THREAD

**Issue by school510587** [nvaccess/nvda#20505]
### Detailed description of the issue

Hi developers,

Under some circumstances, only NVDA 2017.3 along with NVDARemote add-on v2.2 is available for me. However, I find that the add-on failed to connect the nvdaremote.com server for one day. I doesn't find any information that old clients would be dropped from https://github.com/jmdaweb/NVDARemoteServer .

Could you tell me how to get access to the nvdaremote.com server again in the above environment setting?

Please tell me if this issue should be migrated to the issue tracker dedicated for NVDARemote. Thanks a lot for your response!

### Why are the other templates not appropriate in this case?

It is a compatibility issue for an old NVDA version, rather than the latest version.

### Have you asked for advice on how to report this issue via a community discussion? If so, please link to the discussion

None.

### Additional information (if applicable)

If the server doesn't refuse connection from clients using old add-on and old protocol versions, then it is possible that the problem is in SSL/TLS.

### Steps to reproduce or illustrate the issue (if applicable)

Install remote-2.2.nvda-addon in a Windows XP computer with NVDA 2017.3 installed. Select Tools→Remote→Connect... and set the host as nvdaremote.com and set the key as any legal value. After pressing OK, a connection error dialog pops up.

### Expected outcome or behaviour (if applicable)

The old client version on Windows XP can connect to nvdaremote.com successfully.

### Logs, screenshots and other attachments (if applicable)

Nothing can be provided. The add-on doesn't leave error/warning information in the event log.

### NVDA type

installed copy

### NVDA version

2017.3

### Have you tried any other versions of NVDA? If so, please report their behaviors.

NVDA 2024.3.1 with remote-2.6.4.nvda-addon works normally. It can connect to nvdaremote.com without error.

### Windows version

5.1.2600 service pack 3

### Name and version of other software in use when reproducing the issue

None.

### Other information about your system

The original url_handler.exe of remote-2.2.nvda-addon is not executable on Windows XP, but it is comfilable by MSVC 2010. This is also the latest version of Remote Support that is compatible with Windows XP.

**Comment by seanbudd** [comment]
Hi - we do not support add-ons or older versions of NVDA
